### PR TITLE
HA-24 Add pipelines.

### PR DESCRIPTION
**Pull request by fexolm**

### CODE REVIEW

**Comment by fexolm** [comment]
@weeellz please fix travis warnings

File: HaroldAdviser.BL/ICloudInstanceManager.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ﻿using System.Threading.Tasks;
+using HaroldAdviser.Data;
+using HaroldAdviser.ViewModels;
 
 namespace HaroldAdviser.BL
 {
     public interface ICloudInstanceManager
     {
-        Task CreateInstanceAsync();
+        Task CreateInstanceAsync(IWebhook webhook, RepositorySettings settins);
```
Comment (weeellz) [inline HaroldAdviser.BL/ICloudInstanceManager.cs]: typo in settins, should be settings

File: HaroldAdviser.BL/PipelineManager.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+            var pipeline = new Pipeline
+            {
+                CloneUrl = webhook.CloneUrl,
+                Status = PipelineStatus.Webhook,
+                Logs = new List<Log>(),
+                Warnings = new List<Warning>()
+            };
+
+            //TODO: check how lazy inicialization works with insert
```
Comment (weeellz) [inline HaroldAdviser.BL/PipelineManager.cs]: should be initialization

**Review by weeellz — CHANGES_REQUESTED** [review]
fix typos

File: HaroldAdviser.BL/ICloudInstanceManager.cs (hunk 1 above)
Comment (fexolm) [reply HaroldAdviser.BL/ICloudInstanceManager.cs]: fixed

File: HaroldAdviser.BL/PipelineManager.cs (hunk 2 above)
Comment (fexolm) [reply HaroldAdviser.BL/PipelineManager.cs]: fixed

**Review by weeellz — APPROVED** [review]

**Comment by fexolm** [comment]
Branch not stable, please merge it into dev

**Pull request merged**